src/lib_json/json_value.cpp

```diff
@@ -1401,7 +1401,7 @@ void Value::Comments::set(CommentPlacement slot, String comment) {
   if (!ptr_) {
     ptr_ = std::unique_ptr<Array>(new Array());
   }
-  // check comments array boundry.
+  // check comments array boundary.
```

Comment:
Would also need capitalization. Let's just remove the comment.
It's fairly obvious why the if precedes the array element assignment.

Reply:
ok